transforms/code/license_select/ray/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM docker.io/rayproject/ray:2.9.3-py310
+
+# install pytest
+RUN pip install --no-cache-dir pytest
+
+# Copy and install data processing libraries 
+# These are expected to be placed in the docker context before this is run (see the make image).
+COPY --chown=ray:users data-processing-lib-python/ data-processing-lib-python/
+RUN cd data-processing-lib-python && pip install --no-cache-dir -e .
+COPY --chown=ray:users data-processing-lib-ray/ data-processing-lib-ray/ 
+RUN cd data-processing-lib-ray    && pip install --no-cache-dir -e .
+COPY --chown=ray:users python-transform/  python-transform/
+RUN cd python-transform && pip install --no-cache-dir -e .
+
+COPY --chown=ray:users python-transform lc_python
+RUN cd lc_python && pip install .
```

Comment:
Isn't this is a duplication of lines 12 and 13 above (the previous 2 statements)?